src/omero_cli_render.py

```diff
@@ -484,6 +488,23 @@ def set(self, args):
         if 'channels' not in data:
             self.ctx.die(104, "ERROR: No channels found in %s" % args.channels)
 
+        # Previously min/max was used to set the channel window start/end
+        # From version 2 on start/end will be used.
+        if 'version' not in data:
```

Comment:
Up for discussion, the alternative to handling the versioning in the command would be to move this logic up to `ChannelObject.init_from_dict()` i.e. serialize to a unified `ChannelObject` representation independently of the input version.

This might lead us to considerations along the lines of https://github.com/ome/omero-cli-render/pull/15#issuecomment-455526499 i.e. start have versioned loaders.

Reply:
I can write json-schema if it's helpful ;)